packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimeRangeList.tsx

```diff
@@ -96,4 +97,7 @@ const getOptionsStyles = () => ({
     flexGrow: 1,
     alignItems: 'flex-start',
   }),
+  list: css({
+    overflowY: 'scroll',
```

Comment:
I think this'd be `auto`, otherwise the scrollbar will be shown all the time.

Reply:
This list will always overflow, but I'll change to `auto` anyways 😛